.changeset/funny-rockets-speak.md

```diff
@@ -0,0 +1,7 @@
+---
+'hive-console-sdk-rs': patch
+'hive-apollo-router-plugin': patch
+---
+
+Extract Hive Console integration implementation into a new package `hive-console-sdk` which can
```

Comment:
Please also document here the internal changes to the LRU cache implementation, and also the refactor to reduce the Mutex count.